autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -173,10 +180,60 @@ def _get_run_response(self, thread, run):
         """
         while True:
             run = self._wait_for_run(run.id, thread.id)
-            if run.status == "completed":
+            if run.status == "failed":
+                new_messages = []
+                print(f'Run: {run.id} Thread: {thread.id}: failed...')
```

Comment:
If the objective it to log, we should use logger instead of print statements? Same for print statements below.

Reply:
Yup done!